certificate.html

```diff
@@ -19,20 +19,26 @@
       <img src="static/images/logo.svg" alt="Logo" width="220" height="191" />
       <h1 class="mainTitle">Print Your Certificate</h1>
       <div class="divider"></div>
+
+      {{if .Certificate }}
```

Comment:
This block one doesn't seem to end, I get the following:

```
panic: template: certificate.html:132: unexpected EOF

goroutine 1 [running]:
html/template.Must(...)
	/opt/homebrew/Cellar/go/1.21.6/libexec/src/html/template/template.go:368
main.init()
	/Users/mvalais/code/print-your-cert/main.go:100 +0x2f8
exit status 2
```

Reply:
Oh damn, that was a refactoring mistake. I'll fix! Sorry